test/api/mocha/utils/testUtils.js

```diff
@@ -71,6 +151,51 @@ const loadAppData = async (appdataFileName = 'appdata.jsonl') => {
 
 }
 
+/**
+ * Loads all appdata files from the appdata directory, then exports them as JSONL
+ * @param {string} appdataDir - Directory containing appdata files (default: 'appdata')
+ * @returns {Promise<Array>} - Results of processing each file
+ */
+const loadAndExportAllAppData = async (appdataDir = 'appdata') => {
+  const { readdir } = await import('fs/promises')
+
+  const __filename = fileURLToPath(import.meta.url)
+  const __dirname = dirname(__filename)
+  const dirPath = join(__dirname, `../../${appdataDir}`)
+  
+  // Read all files in the directory
+  const files = await readdir(dirPath)
+  const results = []
+  
+  // Process each file
+  for (const file of files) {
+    try {
+      console.log(`Loading ${file}...`)
+      // Load the appdata file
+      await loadAppData(file)
+      
+      console.log(`Exporting data as JSONL...`)
+      // Export data in JSONL format using our utility function
+      const exportedData = await exportAppData({ format: 'jsonl' })
+      
+      // Create output filename
+      const outputPath = join(dirPath, file)
+      
+      // Save to new existing file
+      writeFileSync(outputPath, exportedData, 'utf8')
+      
+      console.log(`Saved to ${outputPath}`)
+      results.push({ file, success: true, outputPath })
+    } 
+    catch (error) {
+      console.error(`Error processing ${file}:`, error)
+      results.push({ file, success: false, error: error.message })
+    }
+  }
+  
+  return results
+}
+
```

Comment:
This exported function is not called by any other test code and should be removed.